src/StackAdmin/AzureStackAdmin/Commands.AzureStackAdmin.Test/Commands.AzureStackAdmin.Test.Netcore.csproj

```diff
@@ -1,47 +0,0 @@
-<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
@deathly809 why are we deleting this file?

Reply:
Module no longer exists.